frontend/src/static/js/main.js

```diff
@@ -112,14 +112,15 @@ window.app = new window.Vue({
 
       this.isTabActive = true;
       this.tabType = tabName;
-
+      this.$store.commit('updateTabState', this.isTabActive);
```

Comment:
Since the tab state in Vuex is always the same with this.isTabActive. We can just remove this.isTabActive and use the one in Vuex.

Reply:
Sure, no problem